index.html

```diff
@@ -1462,6 +1463,9 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
     cancelable Promises</a> used with <code>push()</code> and
     <code>watch()</code>, respectively.
   </p>
+  <p class="note">
```

Comment:
I'm not sure whether this extra information adds something, localhost considered to be secure.